modules/ROOT/pages/rest-sdk/configure-operations.adoc

```diff
@@ -53,7 +74,9 @@ operationIdentifier:
     ]
 ----
 
-Which will take a resource like `/2010-04-01/Accounts/{id}/Messages/SMS/Media.json` leaving it as `/Accounts/{id}/Messages/SMS/Media` improving significantly the naming around the resource (for both XML DSL as well as the UI experience)
+The resource `/2010-04-01/Accounts/{id}/Messages/SMS/Media.json` is now
```

Comment:
this is incorrect, the resource does not change. The resource `/2010-04-01/Accounts/{id}/Messages/SMS/Media.json` is always `/2010-04-01/Accounts/{id}/Messages/SMS/Media.json`. 
What it changes is the operation identifier for the operation that maps to the resource `/2010-04-01/Accounts/{id}/Messages/SMS/Media.json`,